src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -13,6 +17,43 @@
 from WMCore.ReqMgr.DataStructs.RequestError import InvalidStateTransition
 from WMCore.ReqMgr.Tools.cms import releases, architectures
 
+def loadRequestSchema(workload, requestSchema):
+    """
+    _loadRequestSchema_
+    Legacy code to support ops script
+    
+    Does modifications to the workload I don't understand
+    Takes a WMWorkloadHelper, operates on it directly with the schema
+    """
+    schema = workload.data.request.section_('schema')
+    for key, value in requestSchema.iteritems():
+        if isinstance(value, dict) and key == 'LumiList': 
+            value = json.dumps(value)
+        try:            
+            setattr(schema, key, value)
+        except Exception as ex:
+            # Attach TaskChain tasks
+            if isinstance(value, dict) and requestSchema['RequestType'] == 'TaskChain' and 'Task' in key:
```

Comment:
And I tested a few TaskChain and StepChain cases and request creation, approval and clone works fine. Which makes me believe we are not really hitting this Exception here. So it's all fine. Sorry for the false alarm :(